.circleci/config.yml

```diff
@@ -1,5 +1,34 @@
-version: 2.1
+version: 2
 jobs:
+  test_local_docker:
+    working_directory: ~/ckanext-geodatagov
+    machine: true
+    steps:
+      - checkout
+      - run:
+          name: Install Docker Compose
```

Comment:
Docker compose _should_ be already installed so this shouldn't be necessary. Are you seeing something else?

Sorry I missed this when looking at https://github.com/GSA/ckanext-datajson/pull/56/files#diff-1d37e48f9ceff6d8030570cd36286a61R12